nginx-rtmp-module/src/ngx_rtmp_codec_module.c

```diff
@@ -580,30 +580,31 @@ ngx_rtmp_codec_av(ngx_rtmp_session_t *s, ngx_rtmp_header_t *h, ngx_chain_t *in)
     }
 
     /* no conf */
-    if (h->type == NGX_RTMP_MSG_VIDEO
-        && !ngx_rtmp_is_codec_header(ctx->video_codec_id, in))
+    if (!ngx_rtmp_is_codec_header(ctx->video_codec_id, in))
```

Comment:
put the `{` with the same line as the `if` (that's the convention for single line ifs)